### PR TITLE
Update versions for JavaFx Maven Plugin and default JavaFX

### PR DESCRIPTION
**Pull request by pgerhard**
Based on the project generated from https://start.gluon.io/ I updated the versions for the JavaFX Maven Plugin and JavaFX itself

### CODE REVIEW

File: client-archetype-javafx/README.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ The following properties can be customized while creating the project:
 
 | Property                    | Default Value |
 | --------------------------- | ------------- |
-| javafx-version              | 14.0.2.1      |
-| javafx-maven-plugin-version | 0.0.4         |
+| javafx-version              | 16            |
+| javafx-maven-plugin-version | 0.0.5         |
```
Comment (abhinayagarwal) [inline client-archetype-javafx/README.md]: Please update to `0.0.6`

File: client-archetype-javafx/src/main/resources/META-INF/maven/archetype-metadata.xml [hunk 2]
```diff
@@ -2,10 +2,10 @@
 <archetype-descriptor xmlns="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0 http://maven.apache.org/xsd/archetype-descriptor-1.0.0.xsd" name="${rootArtifactId}-parent">
     <requiredProperties>
         <requiredProperty key="javafx-version">
-            <defaultValue>14.0.2.1</defaultValue>
+            <defaultValue>16</defaultValue>
         </requiredProperty>
         <requiredProperty key="javafx-maven-plugin-version">
-            <defaultValue>0.0.4</defaultValue>
+            <defaultValue>0.0.5</defaultValue>
```
Comment (abhinayagarwal) [inline client-archetype-javafx/src/main/resources/META-INF/maven/archetype-metadata.xml]: Please update to `0.0.6`

File: client-archetype-mobile/README.md [hunk 3]
```diff
@@ -26,8 +26,8 @@ The following properties can be customized while creating the project:
 
 | Property                    | Default Value |
 | --------------------------- | ------------- |
-| javafx-version              | 14.0.2.1      |
-| javafx-maven-plugin-version | 0.0.4         |
+| javafx-version              | 16            |
+| javafx-maven-plugin-version | 0.0.5         |
```
Comment (abhinayagarwal) [inline client-archetype-mobile/README.md]: Please update to `0.0.6`

File: client-archetype-mobile/src/main/resources/META-INF/maven/archetype-metadata.xml [hunk 4]
```diff
@@ -2,10 +2,10 @@
 <archetype-descriptor xmlns="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0 http://maven.apache.org/xsd/archetype-descriptor-1.0.0.xsd" name="${rootArtifactId}-parent">
     <requiredProperties>
         <requiredProperty key="javafx-version">
-            <defaultValue>14.0.2.1</defaultValue>
+            <defaultValue>16</defaultValue>
         </requiredProperty>
         <requiredProperty key="javafx-maven-plugin-version">
-            <defaultValue>0.0.4</defaultValue>
+            <defaultValue>0.0.5</defaultValue>
```
Comment (abhinayagarwal) [inline client-archetype-mobile/src/main/resources/META-INF/maven/archetype-metadata.xml]: Please update to `0.0.6`

File: client-archetype-javafx/README.md (hunk 1 above)
Comment (pgerhard) [reply client-archetype-javafx/README.md]: Done

File: client-archetype-javafx/src/main/resources/META-INF/maven/archetype-metadata.xml (hunk 2 above)
Comment (pgerhard) [reply client-archetype-javafx/src/main/resources/META-INF/maven/archetype-metadata.xml]: Done

File: client-archetype-mobile/README.md (hunk 3 above)
Comment (pgerhard) [reply client-archetype-mobile/README.md]: Done

File: client-archetype-mobile/src/main/resources/META-INF/maven/archetype-metadata.xml (hunk 4 above)
Comment (pgerhard) [reply client-archetype-mobile/src/main/resources/META-INF/maven/archetype-metadata.xml]: Done

**Review by abhinayagarwal — APPROVED** [review]

**Pull request merged**